scripts/ci/push-image.sh

```diff
@@ -26,7 +26,7 @@
         beta_prerel_regex="^beta\.[0-9]+"
         prerel="$(semver get prerel "${RELEASE_TAG}" || true)"
         version="$(semver get release "${RELEASE_TAG}" || true)"
-        tags="okteto/okteto:${RELEASE_TAG},okteto/okteto:dev"
+        tags="okteto/okteto:${RELEASE_TAG},okteto/okteto:dev,okteto/okteto:master"
```

Comment:
This looks wrong. `master` should replace the current `latest`, but this is just adding the tag `master`, not replacing anyone